fileshare/libdrop/symbols/cgo.go

```diff
@@ -0,0 +1,9 @@
+package symbols
```

Comment:
Any reason to export this to a separate package when it's always included with
``` go
_ "github.com/NordSecurity/nordvpn-linux/fileshare/libdrop/symbols" // required for linking process
```
in the upper package? I could see the benefit of this somewhere in `main` configuration though as it would allow different linking configuration.

Reply:
I moved it to separate package, because there was a dependency cycle: fileshare needs libdrop to get symbols for linking and libdrop needs fileshare because of `DirDepthLimit`  and `TransferFileLimit` consts